tests/ci/clean_up.sh

```diff
@@ -11,13 +11,11 @@ ID=${ID:-}
 IMAGE_KEY=${IMAGE_KEY:-}
 TEMPDIR=${TEMPDIR:-}
 HTTPD_PATH=${HTTPD_PATH:-}
+DOWNLOAD_DIRECTORY="/var/lib/libvirt/images"
 
 greenprint "🧼 Cleaning up: start"
-# Remove stop and remove the VM
-virsh destroy "${IMAGE_KEY}" || true
-virsh undefine "${IMAGE_KEY}" --nvram
-#TODO: Remove the resulted VM until we find a way to save it at S3
 virsh vol-delete --pool images "${IMAGE_KEY}".qcow2
+rm -f ${DOWNLOAD_DIRECTORY}/${IMAGE_KEY}.raw
```

Comment:
Why don't you use the same method that in the line above? It's more _native_ for dealing with `libvirt` images.
Just copying the line and changing the `.qcow2` to `.raw` will do it. That way you don't need to specify the directory, `libvirt` will know where to look.

Reply:
Thanks, done